tests/new_upgrades/conftest.py

```diff
@@ -98,3 +98,14 @@ def content_upgrade_shared_satellite():
     ) as test_duration:
         yield sat_instance
         test_duration.ready()
+
+
+@pytest.fixture
+def fdi_upgrade_shared_satellite():
+    """Mark tests using this fixture with pytest.mark.content_upgrades."""
```

Comment:
Should be `pytest.mark.discovery_upgrades` right?

Reply:
@shubhamsg199 Since the pre-upgrade scenarios in the new format are fixtures rather than tests, I don't think they should be marked with `*_upgrades` marks. The tests that will be collected and that should be marked in this way are now the post-upgrade scenarios, all of which will inherit their corresponding pre-upgrade scenario fixtures.

Reply:
Yes, I understood I meant the comment of the fixture `"""Mark tests using this fixture with pytest.mark.content_upgrades."""` to `"""Mark tests using this fixture with pytest.mark.discovery_upgrades."""`

Reply:
@shubhamsg199 Whoops, sorry for my misunderstanding. You are right of course. Just made the change.